cmd/update/readme.go

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) arkade author(s) 2022. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package update
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+
+	install "github.com/alexellis/arkade/cmd"
+	system "github.com/alexellis/arkade/cmd/system"
+	"github.com/alexellis/arkade/pkg/get"
+	"github.com/spf13/cobra"
+)
+
+func MakeReadme() *cobra.Command {
+	var command = &cobra.Command{
+		Use:   "readme",
+		Short: "A brief description of your command",
+		Long: `A longer description that spans multiple lines and likely contains examples
+and usage of using your command. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
```

Comment:
Could you include the supported markers in the Cobra help message please?

Reply:
Yes.  Quite the oversight 😆

Reply:
Added some detail here